include/world_builder/world.h

```diff
@@ -275,6 +275,11 @@ namespace WorldBuilder
        */
       std::string interpolation;
 
+      /**
+       * Todo
+       */
+      int local_seed;
```

Comment:
I don't think this is needed. Can you remove this and just locally declare it as an int?